scc/delete_unseen_systems.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Just wondering, is there a reason for this not to be a CronJob instead?

Reply:
No particular reason but cronjobs produces some unneeded workload on the K8s API (triggers the scheduler, etc.) for just this stupid thing :D You can see another approach I've used in the past using cronjobs here https://github.com/e-minguez/openshift-yolo 🤣